integration/airflow/tests/integration/Dockerfile

```
@@ -20,7 +20,7 @@ RUN AIRFLOW_VERSION=`airflow version` && \
     cd ../airflow && \
     pip install --no-cache-dir --use-deprecated=legacy-resolver -r dev-requirements.txt \
     --constraint=https://raw.githubusercontent.com/apache/airflow/constraints-$AIRFLOW_VERSION/constraints-$PYTHON_VERSION.txt
-RUN python -m virtualenv dbt_venv && pwd && source dbt_venv/bin/activate &&  \
+RUN python -m virtualenv dbt_venv && pwd && source dbt_venv/bin/activate && pip install -U pip && \
     unset PIP_USER && \
     cd /app/openlineage/integration/sql && \
     pip install openlineage-sql --no-index --find-links target/wheels --force-reinstall && \
```

```suggestion
RUN python -m virtualenv dbt_venv && pwd && source dbt_venv/bin/activate && \
    unset PIP_USER && \
    python -m pip install pip --upgrade && \
    cd /app/openlineage/integration/sql && \
    pip install openlineage-sql --no-index --find-links target/wheels --force-reinstall && \
```

Seems to do the job for Airflow 2.1.4 integration test build